src/components/RadioButtonWithLabel.js

```diff
@@ -55,13 +58,17 @@ const RadioButtonWithLabel = (props) => {
                     label={props.label}
                     hasError={props.hasError}
                 />
-                <TouchableOpacity
+                <PressableWithFeedback
+                    focusable={false}
+                    accessible={false}
```

Comment:
Why are we setting accessible as false? Don't we want the radio buttons to have accessibility features

Reply:
Here we are referring only to the label next to the radio buton. For now it was only not-focusable (`focusable={false}`) so in web, screen readers cannot access them. Setting the `accessible={false}` will unify this behaviour for native platforms. 
We are not making whole component not accessible, as the radio button component itself is still accessible, making both of them accessible is overkill, as user would have  press tab/scroll the VoiceOver  two times for the same action